mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -169,6 +170,17 @@ CLASS net/minecraft/class_437 net/minecraft/client/gui/screen/Screen
 	METHOD method_40041 hide ([Lnet/minecraft/class_339;)V
 		ARG 0 widgets
 	METHOD method_41843 clearAndInit ()V
+	METHOD method_47412 setTooltips (Lnet/minecraft/class_7919;)V
```

Comment:
@MattiDragon

> > the method must be called every rendering frame
>
> This is true for the generic tooltip system too

You are right. (I didn't notice `MultiplayerScreen` sets the `tooltip` to `null` at the head of `render`.) In that case, I'd rename `MultiplayerScreen#tooltip` to `MultiplayerScreen#multiplayerScreenTooltip` or similar and use singular for the one in `Screen`. It's confusing to use plural just because it conflicts.